### PR TITLE
[729] Get new DRLS branch working with Logica Health - Connectathon

### PR DESCRIPTION
**Pull request by plarocque4**
Change scripts for startup.

### CODE REVIEW

**Comment by lpdecelles** [comment]
> Looks good. Was able to test functionality locally by doing `npm run start`. Running `npm run startDev` confusingly starts up with the proxy configured for the logicahealth.

Pat, was "proxy configured for the logicahealth" one of the changes you needed to make?

**Comment by plarocque4** [comment]
> > Looks good. Was able to test functionality locally by doing `npm run start`. Running `npm run startDev` confusingly starts up with the proxy configured for the logicahealth.
> 
> Pat, was "proxy configured for the logicahealth" one of the changes you needed to make?

Hmm, startDev is assuming that the VERSION environment variable is set, I'll fix that.

I had to configure some things to work with logicahealth, I was trying to streamline the configuration to make it less confusing.

File: package.json [hunk 1]
```diff
@@ -5,10 +5,9 @@
   "scripts": {
     "build": "webpack --config ./webpack.config.prod.js --progress --colors",
     "start": "webpack-dev-server --config ./webpack.config.dev.js & json-server --watch ./src/db.json --port 3006",
-    "startDev": "webpack-dev-server --config ./webpack.config.dev.js",
-    "startProd": "webpack-dev-server --config ./webpack.config.prod.js",
-    "startTemplate": "webpack-dev-server --config ./webpack.config.run.js",
-    "startTest": "node ./setup.js",
+    "startDev": "VERSION=Dev node ./setup.js",
```
Comment (hossenlopp) [inline package.json:8]: Running `npm run startDev` only starts the proxy server and not the webpack-dev-server. Doing `npm run start` seems to start the correct things and is functional in the local development environment. It's all quite confusing to be honest.

Comment (plarocque4) [reply package.json:8]: I'll remove startDev, the start should be enough for the dev case.

**Pull request merged**